instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -422,6 +422,13 @@ def _instrument(self, **kwargs):
                     request.
         """
         lambda_handler = os.environ.get(ORIG_HANDLER, os.environ.get(_HANDLER))
+        if not lambda_handler:
+            raise ValueError(
```

Comment:
Does this raise makes the whole otel distro init fail? If so I don't think we want that. I would just early return and maybe print a warning with `warnings` but I would kinda expect instrumentation to do nothing if they don't have anything to do. 

While you are at it please update the `see more` link in the header to https://github.com/open-telemetry/semantic-conventions/blob/main/docs/faas/aws-lambda.md since the current link is broken

Reply:
Also curious about where this ORIG_HANDLER comes from

Reply:
Hi @xrmx, thanks for your review! It doesn't blocks, in fact before this treatment an `AttributeError` was raised instead, as we can see in [issue](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2744).

Me and @emdneto did some tests to check if this exceptions affect the whole OTEL, but it doesn't. The exception is printed, but the other instrumentations finish succesfully.

Reply:
@xrmx It seems we have had this for a while in specific situations. What I noticed Is the opentelemetry initialization failed and spans/metrics aren't being created

Reply:
@xrmx well, we did more tests and the instrumentation breaks for everyone haha. Based on it, I change the exception for an early return plus a warning. It should release all the other instrumentations and keep tracking about what happened.

Reply:
@xrmx the ORIG_HANDLER comes from [here](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel-instrument#L157)